facebook/__init__.py

```diff
@@ -362,6 +364,18 @@ def debug_access_token(self, token, app_id, app_secret):
         }
         return self.request(self.version + "/" + "debug_token", args=args)
 
+    def get_auth_url(self, app_id, canvas_url, perms=None, **kwargs):
+        """
+        Build link to make login though Facebook Open Authorization (OAuth).
```

Comment:
I think would be better to keep docstring without a link to Facebook Doc, better to keep it at project documentation because it could be changed anytime.
I will take your suggestion of that docstring - "Build a URL to create an OAuth dialog".